packages/matchbox/src/components/Panel/Panel.js

```diff
@@ -35,12 +35,15 @@ const Panel = React.forwardRef(function Panel(props, userRef) {
         position="relative"
         {...innerSystemProps}
         height={innerHeight}
+        bg={appearance === 'inverted' ? 'gray.900' : ''}
```

Comment:
Need to change `border` as well, probably need to keep the border, but change borderColor?